config/types/schema.go

```diff
@@ -0,0 +1,179 @@
+package types
```

Comment:
I was trying to avoid devs needing to have schematyper installed on their systems if they're not changing the spec